client/app/components/packages/landuse-form/change-city-map.hbs

```diff
@@ -1,367 +1,286 @@
-            ) "checked"}}
-          onClick={{fn (mut form.data.dcpEstablishstreetopt)
+            ) "checked"}} onClick={{fn (mut form.data.dcpEstablishstreetopt)
             (if form.data.dcpEstablishstreetopt
               (optionset 'landuseForm' 'dcpEstablishstreetopt' 'code' 'No')
               (optionset 'landuseForm' 'dcpEstablishstreetopt' 'code' 'Yes')
             )
-          }}
-        />
-        <label for="dcpEstablishstreetopt">
-          Street
-        </label>
-
-        <br>
-
-        <Input
-          type="checkbox"
-          id="dcpEstablishparkopt"
-          data-test-checkbox="dcpEstablishparkopt"
-          checked={{if
+          }} />
+    <label for="dcpEstablishstreetopt">
+      Street
+    </label>
+
+    <br>
+
+    <Input type="checkbox" id="dcpEstablishparkopt" data-test-checkbox="dcpEstablishparkopt" checked={{if
             (eq
               form.data.dcpEstablishparkopt
               (optionset 'landuseForm' 'dcpEstablishparkopt' 'code' 'Yes')
-            ) "checked"}}
-          onClick={{fn (mut form.data.dcpEstablishparkopt)
+            ) "checked"}} onClick={{fn (mut form.data.dcpEstablishparkopt)
             (if form.data.dcpEstablishparkopt
               (optionset 'landuseForm' 'dcpEstablishparkopt' 'code' 'No')
               (optionset 'landuseForm' 'dcpEstablishparkopt' 'code' 'Yes')
             )
-          }}
-        />
-        <label for="dcpEstablishparkopt">
-          Park
-        </label>
-
-        <br>
-
-        <Input
-          type="checkbox"
-          id="dcpEstablishpublicplaceopt"
-          data-test-checkbox="dcpEstablishpublicplaceopt"
-          checked={{if
+          }} />
+    <label for="dcpEstablishparkopt">
+      Park
+    </label>
+
+    <br>
+
+    <Input type="checkbox" id="dcpEstablishpublicplaceopt" data-test-checkbox="dcpEstablishpublicplaceopt" checked={{if
             (eq
               form.data.dcpEstablishpublicplaceopt
               (optionset 'landuseForm' 'dcpEstablishpublicplaceopt' 'code' 'Yes')
-            ) "checked"}}
-          onClick={{fn (mut form.data.dcpEstablishpublicplaceopt)
+            ) "checked"}} onClick={{fn (mut form.data.dcpEstablishpublicplaceopt)
             (if form.data.dcpEstablishpublicplaceopt
               (optionset 'landuseForm' 'dcpEstablishpublicplaceopt' 'code' 'No')
               (optionset 'landuseForm' 'dcpEstablishpublicplaceopt' 'code' 'Yes')
             )
-          }}
-        />
-        <label for="dcpEstablishpublicplaceopt">
-          Public Place
-        </label>
-
-        <br>
-
-        <Input
-          type="checkbox"
-          id="dcpEstablishgradeopt"
-          data-test-checkbox="dcpEstablishgradeopt"
-          checked={{if
+          }} />
+    <label for="dcpEstablishpublicplaceopt">
+      Public Place
+    </label>
+
+    <br>
+
+    <Input type="checkbox" id="dcpEstablishgradeopt" data-test-checkbox="dcpEstablishgradeopt" checked={{if
```

Comment:
Sorry to nit, but please revert all these formatting changes since they aren't relevant to the task. The formatting changes also make it hard to tell what are the actual real changes this commit introduces.